### PR TITLE
DOC: fixing spelling & grammar mistakes, providing uniformity in doc strings and comments

### PR DESCRIPTION
**Pull request by glalteva**
Reviewing directories top-down as shown by commits.

### CODE REVIEW

File: datalad/crawler/nodes/annex.py [hunk 1]
```diff
@@ -369,21 +371,21 @@ def __call__(self, data):  # filename=None, get_disposition_filename=False):
...
+            # just add into git directly for now
             # TODO: tune  add so we could use its json output, and may be even batch it
             out_json = _call(self.repo.add, fpath, options=self.options)
         # elif self.mode == 'full':
-        #     # Since addurl ignores annex.largefiles we need first to download that file and then
+        #     # since addurl ignores annex.largefiles we need first to download that file and then
         #     # annex add it
         #     # see http://git-annex.branchable.com/todo/make_addurl_respect_annex.largefiles_option
-        #     lgr.debug("Downloading %s into %s and adding to annex" % (url, filepath))
+        #     lgr.debug("Downloading %s into %s and adding to annex" % url, filepath)
```
Comment (yarikoptic) [inline datalad/crawler/nodes/annex.py]: ;-) then replace % with ,, otherwise comment becomes buggy ;-)

File: datalad/crawler/nodes/annex.py [hunk 2]
```diff
@@ -616,16 +621,16 @@ def merge_branch(self, branch, target_branch=None,
           'theirs' with commit=False can be used to prepare data from that branch for
           processing by further steps in the pipeline
         commit: bool, optional
-          Either to commit when merge is complete
+          Either to commit when merge is complete or not
         one_commit_at_at_time: bool, optional
-          Either to generate
+          Either to generate or not
```
Comment (yarikoptic) [inline datalad/crawler/nodes/annex.py]: I feel we better unity docs for all such flags to be uniform and probable imperative in most cases. @hanke what do you think? Also worth comparing to language  used in other popular toolboxes, eg numpy, nibabel, pandas

If imperative, In this case should be

Merge each commit of the source branch one at a time. Might be necessary to e.g. postprocess merged content for each state of the source branch

File: datalad/crawler/nodes/annex.py [hunk 3]
```diff
@@ -988,11 +999,12 @@ def finalize(self, tag=False, existing_tag=None, cleanup=False):
           Also `last_version` is provided as the last one from stats.versions (None
           if empty)
         existing_tag: None or '+suffix', optional
-          What to do if tag already exists, if None -- warning is issued. If `+suffix`
+          What to do if tag already exists, if None:\, warning is issued. If `+suffix`,
```
Comment (yarikoptic) [inline datalad/crawler/nodes/annex.py]: Hm, is this some day construct I don't know?

File: datalad/crawler/nodes/annex.py [hunk 4]
```diff
@@ -1035,7 +1048,8 @@ def _finalize(data):
                             tag_ = tag__
                         else:
                             raise ValueError(existing_tag)
-                    self.repo.repo.create_tag(tag_, message="Automatically crawled and tagged by datalad %s.%s" % (__version__, stats_str))
+                    self.repo.repo.create_tag(tag_, message="Automatically crawled and tagged by datalad %s.%s" % (
+                    __version__, stats_str))
```
Comment (yarikoptic) [inline datalad/crawler/nodes/annex.py]: Not sure if I like such unindented formating

File: datalad/crawler/nodes/misc.py [hunk 5]
```diff
@@ -363,10 +362,10 @@ def func_node(data):
                 yield data
 
     in_args = list(_string_as_list(data_args)) + list(_string_as_list(data_kwargs))
-    func_node.__doc__ = """Function %s wrapped into a node
+    func_node.__doc__ = """function %s wrapped into a node.
```
Comment (yarikoptic) [inline datalad/crawler/nodes/misc.py]: This will be the opening of the docstring... we have those capitalized

**Comment by yarikoptic** [comment]
Thank you Gergana!

File: datalad/crawler/nodes/annex.py (hunk 3 above)
Comment (glalteva) [inline datalad/crawler/nodes/annex.py]: Nope, this was me trying to delete `:` but pressing the backslash and still managing to add the comma! It'll be fixed.

**Comment by yarikoptic** [comment]
you weren't alone in doing wide changes.  so now we have merge conflicts and it might be tricky to resolve -- let me know if you need help ;)

**Pull request merged**